### PR TITLE
Add google analytics

### PR DESCRIPTION
**Pull request by SteveBarnett**

### CODE REVIEW

**Comment by SteveBarnett** [comment]
Ready for review. @hodgestar can you spare an eyeball please?

File: go/templates/app.html [hunk 1]
```diff
@@ -100,3 +100,15 @@
...
 {% endblock %}
+
+{% block googleanalytics %}
+<script>
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+  ga('create', 'UA-36658813-1', 'auto');
```
Comment (hodgestar) [inline go/templates/app.html]: We shouldn't hardcode the Google Analytics key -- we should pull it out of the template context, e.g.:

```
{% if google_analytics_ua %}
<script>
    ...
</script>

ga('create', '{{ google_analytics_ua }}', 'auto');
{% endif %}
```

The context processor already exists in `go/base/context_processors.py`.

File: go/tests/test_views.py [hunk 2]
```diff
@@ -70,3 +70,14 @@ def test_help_not_authenticated(self):
...
+
+
+class TestApp(GoDjangoTestCase):
+    def setUp(self):
+        self.vumi_helper = self.add_helper(DjangoVumiApiHelper())
+
+    def test_google_analytics(self):
+        client = self.vumi_helper.get_client('superuser@example.com')
+        response = client.get(reverse('home'), follow=True)
+        self.assertContains(response, 'UA-36658813-1')
```
Comment (hodgestar) [inline go/tests/test_views.py]: The test failed. I'm not entirely sure why*. We can add a dummy GOOGLE_ANALYTICS_UA key to `go/testsettings.py` once we're using the value provided by the context processor.

_Well, the response returned didn't contain the GA key, but I'm not sure why *that_ was.

**Comment by hodgestar** [comment]
Otherwise looks good. :)

**Comment by SteveBarnett** [comment]
:+1: , happy with the @hodgestar changes.

**Comment by SteveBarnett** [comment]
Fixes #1006 .

**Pull request merged**